plugins/base/src/main/kotlin/translators/psi/PsiPackageHelper.kt

```diff
@@ -0,0 +1,15 @@
+/**
+ * Pretend to be from `com.intellij.psi` in order to access package-private declarations
+ */
+@file:Suppress("PackageDirectoryMismatch")
+package com.intellij.psi
+
+import com.intellij.lang.jvm.annotation.JvmAnnotationAttributeValue
+
+/*
+ * For some reason `PsiAnnotationConstantValue` is package private,
```

Comment:
Can you use a public interface `JvmAnnotationConstantValue` instead of `PsiAnnotationConstantValue`?

Reply:
Wow, yeah, totally. 

I was looking for something like this, but when you open up `PsiAnnotationConstantValue` and go to declaration of `JvmAnnotationConstantValue` from the signature - it goes to the kotlin-compiler artifact, where the interface is empty, so I thought there was no other way :facepalm: 

Importing it seems to be fine though

![2022-08-03_15-04-12](https://user-images.githubusercontent.com/22685910/182593559-676db115-7b2c-4d41-883d-3df666ae0ed2.png)